src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -48,6 +48,48 @@
  * &lt;module name=&quot;DefaultComesLast&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   void method(int i) {
+ *     switch (i) {
+ *       case 1:
+ *         break;
+ *       case 2:
+ *         break;
+ *       default:
+ *         // code
+ *         break; // OK
+ *     }
+ *
+ *     switch (i) {
+ *       case 1:
+ *         break;
+ *       case 2:
+ *         break; // OK
+ *     }
+ *
+ *     switch (i) {
+ *       case 1:
+ *         break;
+ *       default: // violation 'default' before 'case'
+ *         break;
+ *       case 2:
+ *         break;
+ *     }
+ *
+ *     switch (i) {
+ *       case 1:
+ *         default: // violation 'default' before 'case'
+ *           break;
+ *       case 2:
+ *         break;
+ *     }
+ *   }
+ * }
+ * </pre>
+ * <p>
  * To configure the check for skipIfLastAndSharedWithCase:
```

Comment:
`To configure the check for skipIfLastAndSharedWithCase` -> 
To configure the check to allow default label to be not last if it is shared with case

Reply:
done.